src/actions/action.js

```diff
@@ -0,0 +1,5 @@
+export const LOGIN_SUCCESS = 'LOGIN_SUCCESS';
+export const LOGIN_FAILURE = 'LOGIN_FAILURE';
+
+export const REGISTER_SUCCESS = 'REGISTER_SUCCESS';
+export const REGISTER_FAILURE = 'REGISTER_FAILURE';
```

Comment:
For a improvement I will add another  action type like `FETCHING_START ` so you you can keep track when your action request as started so you can implement a easy Spinner maybe